### DISCUSSION TITLE
Add the ability to use saved replies

### DISCUSSION THREAD

**Discussion by ErikEJ** [community/community#62758] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

That is in many cases the onlyreason why I have to go back to the web UI.

**Comment by Iron-Ham** [comment, marked as answer]
@ErikEJ I'm happy to report that saved replies are already in the app.

You can access them in a comment via the markdown bar: scroll all the way to the right and tap on the rightmost icon. This will bring up a modal which has all your previously saved replies for quick access.

**Reply by ErikEJ** [reply]
That is wonderful but not very easy to discover. Thanks!

**Reply by jsoref** [reply]
As noted, it isn't remotely easy to discover. I've been using this app on and off for a long time and only recently discovered that the more in the toolbar. 

It'd take me too long to look up the Android style guide reference, but I'm reasonably confident that side scrolling toolbars are not recommended. 

None of the Google apps I have use anything like this. 

The closest is in the Photos editor:
![Screenshot_20240721-161315.png](https://github.com/user-attachments/assets/fe158e52-5189-492c-be03-d0edda2a292b)

Note that it's using a button that's clearly clipped in the top row and text in the second row where the first item is centered and the second item is too close to the right edge for someone to think that's the right edge of the bar.
